HW8.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<h3> 1. Вивести на сторінку в один рядок через кому числа від 10 до 20. </h3>
+<p id="1"></p>
+
+<h3> 2. Вивести квадрати чисел від 10 до 20. </h3>
+<p id="2"></p>
+
+<h3> 3. Вивести таблицю множення на 7. </h3>
+<p id="3"></p>
+
+<h3> 4. Знайти суму всіх цілих чисел від 1 до 15. </h3>
+<p id="4"></p>
+
+<h3> 5. Знайти добуток усіх цілих чисел від 15 до 35. </h3>
+<p id="5"></p>
+
+<h3> 6. Знайти середнє арифметичне всіх цілих чисел від 1 до 500. </h3>
+<p id="6"></p>
+
+<h3> 7. Вивести суму лише парних чисел в діапазоні від 30 до 80. </h3>
+<p id="7"></p>
+
+<h3> 8. Вивести всі числа в діапазоні від 100 до 200 кратні 3. </h3>
+<p id="8"></p>
+
+<h3> 9. Дано натуральне число (яке ви ввели). Знайти та вивести на сторінку всі його дільники. </h3>
+<p id="9"></p>
+
+<h3> 10. Визначити кількість його парних дільників. </h3>
+<p id="10"></p>
+
+<h3> 11. Знайти суму його парних дільників. </h3>
+<p id="11"></p>
+
+<h3> 12. Надрукувати повну таблицю множення від 1 до 10. </h3>
+<p id="12"></p>
+
+<script>
+    //Вивести на сторінку в один рядок через кому числа від 10 до 20.
+    let text1 = " ";
```

Comment:
To isolate each subtask you can use the IIFE function or block
```
// Block
{
 // here is the subtask
}

// IIFE
(() => {
 // here is the subtask
})()
```

Reply:
Updated